ros2pkg/ros2pkg/resource/CMakeLists.txt.em

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.5)
+project(@(project_name))
+
+@[for deb in dependencies]find_package(@deb REQUIRED)
+@[end for]
```

Comment:
For ease of readability this should be formatted like this:

```
@[for deb in dependencies]@
find_package(@deb REQUIRED)
@[end for]@
```

Note the trailing `@` to avoid a newline.

Same below.